lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -515,6 +523,28 @@ walletId =
 --
 -- Helpers
 --
+genMnemonics
+   :: forall mw ent csz.
+       ( ConsistentEntropy ent mw csz
+       , ValidEntropySize ent
+       , ValidChecksumSize ent csz
+       , ent ~ EntropySize mw
+       , mw ~ MnemonicWords ent
+       )
+   => IO [Text]
+genMnemonics =
+    mnemonicToText . entropyToMnemonic @mw <$> genEntropy
+
+data MnemonicLength = M9 | M12 | M15 | M18 | M21 | M24 deriving (Show)
+
+genMnemonics' :: MnemonicLength -> IO [Text]
```

Comment:
Small stylistic note, I usually name things the other way around: the function that is going to be the most commonly used without `'` or any mark, and the internal / advanced one with an uglier name. But, no strong feeling :shrug: